### PR TITLE
node pg upgrade #27 

### PR DESCRIPTION
**Pull request by naazy**
node pg upgrade #27 as per https://node-postgres.com/guides/upgrading

### CODE REVIEW

**Comment by Jbarget** [comment]
@naazy its talkin' about this line: https://github.com/dwyl/hapi-postgres-connection/blob/pgConnectDegradation/test/server_example.js#L49

apparently `result` is undefined

**Comment by naazy** [comment]
Thanks @Jbarget will look into this later

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "hapi-postgres-connection",
-  "version": "6.1.0",
+  "version": "6.1.1",
```
Comment (nelsonic) [inline package.json]: @naazy are you upgrading the version of `node-pg` in `package.json > dependencies` ?

**Review by nelsonic — APPROVED** [review]
@naazy looks good _so far_, but are you upgrading the dependency on `node-pg` ?

**Comment by naazy** [comment]
@nelsonic have upped node-pg and changed the version of this module to 6.2.0

**Review by nelsonic — APPROVED** [review]
@naazy thanks for updating! 👍

**Comment by nelsonic** [comment]
@naazy latest version `hapi-postgres-connection@6.2.0` is on NPM! 
Thanks again for making time to do these updates! 🎉

**Pull request merged**